src/script/api/script_list.hpp

```diff
@@ -88,11 +88,9 @@ class ScriptList : public ScriptObject {
 			sq_push(vm, 2);
 		}
 
-		/* Don't allow docommand from a Valuator, as we can't resume in
+		/* Don't allow docommand from a filter, as we can't resume in
 		 * mid C++-code. */
-		bool backup_allow = ScriptObject::GetAllowDoCommand();
-		ScriptObject::SetAllowDoCommand(false);
-
+		ScriptObject::DisableDoCommandScope disabler{};
```

Comment:
This syntax caused an early end of class detection by the exporter, and skipped all the public class members appearing after.

I fixed the exporter.